compiler/il/OMRNode.hpp

```diff
@@ -533,6 +533,7 @@ class OMR_EXTENSIBLE Node
    /// and the opcodes for highOp/adjunctOp are luaddc/luadd, or lusubb/lusub.
    ///
    bool                   isTernaryHigh();
+   bool                   isSelectHigh();
```

Comment:
The doxygen should be on isSelectHigh with a deprecated warning on isTernaryHigh.

Reply:
Thank you for your comment, I am updating this for other similar instances.